deploy.go

```diff
@@ -260,7 +260,10 @@ for updates to this repository's remote master branch.`,
 			if err != nil {
 				log.Fatal(err)
 			}
-			repoName := common.ExtractRepository(common.GetSSHRemoteURL(origin.Config().URLs[0]))
+			repoName, err := common.ExtractRepository(common.GetSSHRemoteURL(origin.Config().URLs[0]))
+			if err != nil {
+				log.Fatal(err)
```

Comment:
While the error check is good, I'm a bit iffy on `ExtractRepository` blocking remote setup since it's a purely cosmetic thing (used to generate direct links to repo configuration post-bootstrap).

Maybe we could have some kind of warning instead, and assign `repoName = $YOUR_REPOSITORY` (so that the post-bootstrap output will print something like `"go to https://github.com/$YOUR_REPOSITORY/settings"` )